packages/client3/.eslintrc.json

```diff
@@ -3,7 +3,7 @@
   "extends": ["next/core-web-vitals", "plugin:storybook/recommended", "plugin:@typescript-eslint/recommended"],
   "plugins": ["@typescript-eslint"],
   "rules": {
-    "no-console": ["error", { "allow": ["warn", "error"] }],
+    "no-console": ["warn", { "allow": ["warn", "error"] }],
```

Comment:
Up to the group, but it was tripping me out for when I was debugging and getting errors for `console.logs`

Reply:
Agreed, arbys is the same by the way - https://github.com/weavik/arbys-bootstrap/blob/main/web/.eslintrc.js#L79